sdks/java/io/elasticsearch-tests/elasticsearch-tests-2/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTest.java

```diff
@@ -193,6 +193,19 @@ public void testWritePartialUpdate() throws Exception {
     elasticsearchIOTestCommon.testWritePartialUpdate();
   }
 
+  @Test
+  public void testWritePartialUpdateWithErrors() throws Exception {
```

Comment:
Please, try to stick to the existing test structure. Do not pass the configuration or use special types and index (there is a protection on index name for concurrency that you lose in your test). Use the existing configuration. I would suggest that you rewrite this test and the ones for the other ES versions that way:
``
  public void testWritePartialUpdateWithErrors() throws Exception {
  elasticsearchIOTestCommon.setPipeline(pipeline);
    elasticsearchIOTestCommon.testWritePartialUpdateWithErrors();
  }
``
and in the actual test do:
``
 elasticsearchIOTestUtils.setIndexMapping();
.... test .....
elasticsearchIOTestUtils.unsetIndexMapping();
``

Reply:
If i have set the index mapping, the settings cannot be unset and could cause other problems. So, set the index on existed index, may cause unexpected problems. That's the reason why i use special types and index.